napari/layers/image/image.py

```diff
@@ -486,6 +499,23 @@ def rendering(self, rendering):
         self._rendering = Rendering(rendering)
         self.events.rendering()
 
+    @property
+    def render_as_plane(self):
+        """Should this layer be rendered as a plane.
+        If False, render as volume (default). If True, render as a plane.
+        """
+        return self._render_as_plane
+
+    @render_as_plane.setter
+    def render_as_plane(self, value: bool):
+        if self._render_as_plane is not value:
+            self._render_as_plane = value
+            self.events.render_as_plane()
+
+    @property
+    def plane(self):
```

Comment:
Sorry if I missed it (I'm on mobile), but were you planning to have a setter for `Image.plane`? As I mentioned above, I think we should accept Plane3D objects and dictionaries that can be passed to Plane3D.

Reply:
Agreed, I'm ok with this coming in a follow-up PR though